config/deploy.rb

```diff
@@ -12,6 +12,9 @@
 # Default deploy_to directory is /var/www/my_app_name
 set :deploy_to, '/opt/app/rialto/rialto'
 
+# Set the honeybadger env to match the capistrano stage
+set :honeybadger_env, fetch(:stage)
```

Comment:
This is only going to set the environment when you're deploying. I think the change you want to make is something like this: https://github.com/sul-dlss/puppet/pull/3731

Reply:
@jcoyne Cool, thanks! https://github.com/sul-dlss/puppet/pull/3791

That means we can close this PR, correct?

Reply:
yes.

Reply:
Actually, maybe not.  This will affect the version reported in the deploy.  I don't think we're doing that right currently: https://app.honeybadger.io/projects/58393/deploys